src/helpers/formatAmount.ts

```diff
@@ -4,133 +4,151 @@ import { pipe } from './pipe';
 import { DECIMALS, DIGITS, ZERO } from '../constants';
 import { stringIsInteger } from './stringIsInteger';
 
-export interface FormatAmountPropsType {
-  addCommas?: boolean;
+export interface FormatAmountProps {
+  /**
+   * The raw integer amount (string) in the smallest token unit.
+   * Must be a valid integer string (no decimals, may include leading "-").
+   */
+  input: string;
+  /**
+   * Number of decimals defined by the token (e.g. 18 for ERC-20 ETH).
+   * Default: DECIMALS (typically 18).
+   */
   decimals?: number;
+  /**
+   * Maximum number of decimal digits to display.
+   * Combined with `showLastNonZeroDecimal` to decide padding or trimming.
+   * Default: DIGITS (typically 4).
+   */
   digits?: number;
-  input: string;
+  /**
+   * If true, insert thousands separators (commas) into the integer part.
+   * Default: false.
+   */
+  addCommas?: boolean;
+  /**
+   * If true, amounts smaller than the smallest displayable unit will show
+   * a "<0.00...1" format instead of "0.00...0".
+   * Default: false.
+   */
   showIsLessThanDecimalsLabel?: boolean;
+  /**
+   * Controls decimal formatting behavior:
+   * - If true: pads or trims decimals to exactly `digits` places
```

Comment:
Controls decimal formatting behavior. Example: 
actual value: 0.120009
with `digits` set to 2, displayed value will be: 0.12
with `digits` set to 2 and `showLastNonZeroDecimal`, displayed value will be actual value: 0.120009